src/hooks/useLoadDisciplines.ts

```diff
@@ -0,0 +1,40 @@
+import useLoadFromEndpoint, { ReturnType } from './useLoadFromEndpoint'
+import { DisciplineType, ENDPOINTS } from '../constants/endpoints'
+
+interface ServerResponse {
+  id: number
+  title: string
+  description: string
+  icon: string
+  total_training_sets: number
+  total_discipline_children: number
+  total_documents: number
+}
+
+const formatServerResponse = (serverResponse: ReturnType<ServerResponse[]>): ReturnType<DisciplineType[]> => {
+  const formattedServerResponse: DisciplineType[] =
+    serverResponse.data?.map(item => ({
+      ...item,
+      numberOfChildren:
+        item.total_discipline_children > 0
+          ? item.total_discipline_children
+          : item.total_training_sets > 0
+          ? item.total_training_sets
+          : item.total_documents > 0
+          ? item.total_documents
+          : 0,
```

Comment:
Isn't this the same as:
```suggestion
        item.total_discipline_children ||  item.total_training_sets ||  item.total_documents
```

Reply:
yes it is, and so much nicer :+1: